cli/internal/run/run_test.go

```diff
@@ -81,6 +84,22 @@ func TestParseConfig(t *testing.T) {
 			},
 			[]string{"foo"},
 		},
+		{
+			"concurrency percent",
+			[]string{"foo", "--concurrency=100%"},
+			&Opts{
+				runOpts: runOpts{
+					concurrency: cpus,
+				},
+				cacheOpts: cache.Opts{
+					Dir:     defaultCacheFolder,
+					Workers: 10,
```

Comment:
Relatedly, we don't even give you a choice here, which should probably be mapped to the same calculated concurrency value:
https://github.com/vercel/turborepo/blob/2419e2fc98a04b698be98e54aae9536498fa15ec/cli/internal/config/config.go#L210

Reply:
Agreed, we should wire it up at some point. It also probably should not live in a top-level config, since it's not relevant to every command.